core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -43,31 +44,45 @@ <h1 class="stat-value"><[dashboardStats.total_plays]></h1>
           <p class="stat-description">Open feedback</p>
           <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
         </div>
+        <div class="total-open-feedback stats-card">
+          <p class="stat-description">Subscribers</p>
+          <h1 class="stat-value"><[subscribersList.length]></h1>
+        </div>
       </md-card>
     </div>
 
-    <div ng-if="explorationsList.length > 0 || collectionsList.length > 0">
+    <div ng-if="explorationsList.length > 0 || collectionsList.length > 0 || subscribersList.length > 0">
       <ul class="oppia-dashboard-tabs">
         <li ng-class="{'oppia-dashboard-tabs-active': activeTab === 'myExplorations'}">
           <a class="oppia-dashboard-tabs-text"
              ng-click="setActiveTab('myExplorations')"
-             translate="I18N_DASHBOARD_MY_EXPLORATIONS">
+             translate="I18N_DASHBOARD_EXPLORATIONS">
           </a>
         </li>
         {% if not can_create_collections %}
           <!-- A layout fix when the 'My Collections' tab is absent -->
```

Comment:
Er ... yep, I understand. I just meant, add a full stop (period) to the end of the existing comment. :)